pkg/mounter/ossfs.go

```diff
@@ -53,7 +55,13 @@ func NewFuseOssfs(configmap *corev1.ConfigMap, m metadata.MetadataProvider) Fuse
 				registry = defaultRegistry
 			}
 		}
-		config.Image = fmt.Sprintf("%s/acs/csi-ossfs:%s", registry, defaultOssfsImageTag)
+		tag := defaultOssfsImageTag
+		// if enabled UpdatedOssfsVersion featuregate
+		if features.FunctionalMutableFeatureGate.Enabled(features.UpdatedOssfsVersion) {
```

Comment:
only when UpdatedOssfsVersion feature-gate is enabled, then defaultOssfsUpdatedImageTag will be used.
we keep 2 'default' image tag, for ossfs 1.88.x and 1.91.x.

Reply:
@mowangdk